src/media/media.service.ts

```diff
@@ -21,43 +21,48 @@ export class MediaService {
   ) {}
 
   async uploadMenuItemImage(image: Express.Multer.File, user: AuthUser) {
-    const originalImage = await this.sharpService.resize(
+    const originalImage = await this.uploadImageToCloud(
```

Comment:
you can do map here. 
`[this.ORIGINAL_IMAGE_RESOLUTION, this.MEDIUM_THUMBNAIL_IMAGE_RESOLUTION, ...].map(resolution => {})`